deployment/helm/ditto/values.yaml

```diff
@@ -219,6 +219,12 @@ ingress:
         backendSuffix: gateway
     # annotations defines k8s annotations to add to the Ingress
     annotations:
+      # type of authentication
+      nginx.ingress.kubernetes.io/auth-type: basic
+      # name of the secret that contains the user/password definitions
+      nginx.ingress.kubernetes.io/auth-secret: my-ditto-nginx-ingress-htpasswd
```

Comment:
`my-ditto` is your release name, right?
That should not be a part of the values.yaml
Do you have another idea of how to solve this?